internal/cobraext/flags.go

```diff
@@ -65,6 +65,18 @@ const (
 	BenchCorpusRallyUseCorpusAtPathFlagName        = "use-corpus-at-path"
 	BenchCorpusRallyUseCorpusAtPathFlagDescription = "path of the corpus to use for the benchmark: if present no new corpus will be generated"
 
+	BenchStreamBackFillFlagName        = "backfill"
+	BenchStreamBackFillFlagDescription = "amount of time to ingest events for since starting from now: expressed as a negative duration (ie: -15m)"
```

Comment:
I was thinking about making this -15m by default as the default when you go to Discover is "last 15 min".